frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -862,7 +857,8 @@ private static function sendNotificationEmail(
             'email' => $email,
             'name' => $username,
```

Comment:
```suggestion
            'name' => $name,
```

puedes de paso cambiar esto para evitar la tentación en el futuro de usar esto como username?

Reply:
Listo.